docs/getting-started/platform-setup.md

```diff
@@ -188,32 +199,49 @@ v1.7.3 using:
 
 2. Configure MetalLB using:
 
-    !!! attention
-        Replace the `<i>` in the below snippet with a value between `1` and
-        `254`. Choose a different value for each of your clusters. For
-        conveniently deploying Carbyne Stack as described in the
-        [Deployment Guide](../deployment/), consider using `1` and `2`,
-        respectively.
-
-    ```shell
-    export SUBNET=172.18.<i>.255/25
-    cat <<EOF | envsubst > metallb.yaml
-    apiVersion: v1
-    kind: ConfigMap
-    metadata:
-      namespace: metallb-system
-      name: config
-    data:
-      config: |
-        address-pools:
-        - name: default
-          protocol: layer2
-          addresses:
-          - ${SUBNET}
-          avoid-buggy-ips: true
-    EOF
-    kubectl apply -f metallb.yaml
-    ```
+    === "Apollo"
+
+        ```shell
+        export SUBNET=172.18.1.255/25
+        cat <<EOF | envsubst > metallb.yaml
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+          namespace: metallb-system
+          name: config
+        data:
+          config: |
+            address-pools:
+            - name: default
+              protocol: layer2
+              addresses:
+              - ${SUBNET}
+              avoid-buggy-ips: true
+        EOF
+        kubectl apply -f metallb.yaml
+        ```
+
+    === "Starbuck"
+
+        ```shell
+        export SUBNET=172.18.2.255/25
+        cat <<EOF | envsubst > metallb.yaml
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+          namespace: metallb-system
+          name: config
+        data:
+          config: |
+            address-pools:
+            - name: default
+              protocol: layer2
+              addresses:
+              - ${SUBNET}
+              avoid-buggy-ips: true
+        EOF
+        kubectl apply -f metallb.yaml
+        ```
 
 3. Export the external IP assigned to the Istio Ingress Gateway for later use:
```

Comment:
Since it can take some time until the IP of the Istio ingress gateway can be queried, it would be advantageous to include a step for verification.
The inexperienced user will easily run into problems during setup. -  We missed this in the previous version